app/chitter.rb

```diff
@@ -0,0 +1,29 @@
+require 'sinatra/base'
+require 'data_mapper'
+require 'rack-flash'
+require 'sinatra/partial'
+require_relative 'data_mapper_setup'
```

Comment:
If it is removed the database is not setup and datamapper does not have an 'adapter' to work with?